<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>685</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>685</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QTableWidget" name="fileTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="rowCount">
      <number>4</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QTableWidget" name="commandTable">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <item row="0" column="0">
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="uploadPersonList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>بارگذاری اطلاعات با هدر</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setMouseClick">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>تنظیم کلیک موس</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="typeOneColumnTextFromList">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>تایپ یک ستون از لیست</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="typeMultiColumnTextFromList">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>تایپ چند ستون از لیست</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="typeCustomText">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>تایپ متن دلخواه</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setDelay">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>تنظیم تاخیر</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveUpCommand">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>بالا بردن دستور</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveDownCommand">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>پایین آوردن دستور</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeCommand">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>حذف کردن دستور</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeAllCommand">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>حذف تمام دستورات</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stratProccess">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IRANSansMobile</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>شروع فرآیند</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QFrame" name="actionDialog">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(224, 244, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="commandSetCancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>115</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>IRANSansMobile</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 60, 46)</string>
        </property>
        <property name="text">
         <string>لغو دستور</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="commandSetEnd">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>115</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>IRANSansMobile</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 127);</string>
        </property>
        <property name="text">
         <string>اتمام دستور</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="actionDialogLabel">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>IRANSansMobile</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>متن دلخواه خود را وادر نمایید</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
